resources/build.sh

```diff
@@ -15,5 +15,6 @@ builder_parse "$@"
 
 if builder_start_action test; then
   ./build/test/test.sh
+  ./build/publish-minimum-versions.sh test
```

Comment:
Does this belong here? I thought `/resources/build.sh test` was for unit testing the builder scripts.

Maybe the GHA calls ./build/publish-minimum-versions.sh test?

Reply:
Yes, this does belong here. `resources/build.sh test` should call _all_ the testable objects under /resources (currently, that's only shell scripts -- builder, and publish-minimum-versions.sh script, but that may change in the future).